test/plugin/test_out_google_cloud.rb

```diff
@@ -117,9 +118,9 @@ def setup
 
   # rubocop:disable Metrics/LineLength
   PRIVATE_KEY_CONFIG = %(
```

Comment:
on line 1215 there's a http stub with a comment "Used for 'private_key' auth." which I presume we can remove (sorry, github won't let me comment on the actual line).